core/services/beacon/main.py

```diff
@@ -53,7 +56,7 @@ def __repr__(self) -> str:
 class Beacon:
     def __init__(self) -> None:
         self.runners: Dict[str, AsyncRunner] = {}
-        self.manager = Manager("Beacon", SettingsV1)
+        self.manager = Manager(SERVICE_NAME, SettingsV1)
```

Comment:
The service name is not the same as before, can that be a problem ?

Reply:
No. Talked to @Williangalvani about that and he approved the change.